app/src/main/res/layout/faq_single_activity.xml

```diff
@@ -62,7 +62,7 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:fontFamily="sans-serif-medium"
-            android:textColor="@color/oppia_primary_text"
+            android:textColor="@color/component_color_faq_activity_text_color"
```

Comment:
@bhaktideshmukh, in the FAQ activity the text color is not correct, according to the mocks it should be **#C4C4C4** not def white, PTAL.

Reply:
@Akshatkamboj14 Do we need to consider the mock colors or the colors which are presently used in develop branch for light mode?
Currently in develop branch def_white color is used for textColor.
@rt4914 please confirm whether the mock colors should be used or colors of develop branch.

Reply:
By Looking at different mocks I can confirm that `#C4C4C4` is incorrect in mocks. That colour is used for secondary text but here these texts are primary and therefore current color is right.